2024-04-03 01:46:29,821 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2024-04-03 01:46:48,762 (INFO) - playground.views - Calling httpbin
2024-04-03 01:46:51,521 (INFO) - playground.views - Received the response
2024-04-03 01:47:49,097 (INFO) - django.utils.autoreload - /Users/ermiyasggod/Desktop/DjangoProject/storefront3/storefront/settings.py changed, reloading.
2024-04-03 01:47:50,765 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2024-04-03 01:48:30,319 (INFO) - playground.views - Calling httpbin
2024-04-03 01:48:33,220 (INFO) - playground.views - Received the response
2024-04-03 01:49:30,135 (INFO) - playground.views - Calling httpbin
2024-04-03 01:49:32,875 (INFO) - playground.views - Received the response
2024-04-03 02:12:12,062 (INFO) - django.utils.autoreload - /Users/ermiyasggod/Desktop/DjangoProject/storefront3/manage.py changed, reloading.
2024-04-03 02:16:11,989 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2024-04-03 02:16:20,823 (INFO) - playground.views - Calling httpbin
2024-04-03 02:16:23,592 (INFO) - playground.views - Received the response
2024-04-03 02:18:28,589 (WARNING) - django.request - Unauthorized: /store/collections/
2024-04-03 02:18:28,613 (WARNING) - django.request - Forbidden: /store/collections/
2024-04-03 02:18:28,631 (WARNING) - django.request - Bad Request: /store/collections/
2024-04-03 02:22:30,982 (INFO) - django.utils.autoreload - /Users/ermiyasggod/Desktop/DjangoProject/storefront3/storefront/settings/common.py changed, reloading.
2024-04-03 02:22:32,660 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2024-04-03 02:22:37,316 (INFO) - django.utils.autoreload - /Users/ermiyasggod/Desktop/DjangoProject/storefront3/storefront/settings/common.py changed, reloading.
2024-04-03 02:23:22,260 (WARNING) - django.request - Unauthorized: /store/collections/
2024-04-03 02:23:22,287 (WARNING) - django.request - Forbidden: /store/collections/
2024-04-03 02:23:22,312 (WARNING) - django.request - Bad Request: /store/collections/
2024-04-03 02:25:46,257 (WARNING) - django.request - Unauthorized: /store/collections/
2024-04-03 02:25:46,274 (WARNING) - django.request - Forbidden: /store/collections/
2024-04-03 02:25:46,282 (WARNING) - django.request - Bad Request: /store/collections/
2024-04-03 02:28:14,459 (WARNING) - django.request - Unauthorized: /store/collections/
2024-04-03 02:28:14,476 (WARNING) - django.request - Forbidden: /store/collections/
2024-04-03 02:28:14,485 (WARNING) - django.request - Bad Request: /store/collections/
2024-04-03 02:29:55,100 (WARNING) - django.request - Unauthorized: /store/collections/
2024-04-03 02:29:55,113 (WARNING) - django.request - Forbidden: /store/collections/
2024-04-03 02:29:55,123 (WARNING) - django.request - Bad Request: /store/collections/
2024-04-03 02:30:13,900 (WARNING) - django.request - Unauthorized: /store/collections/
2024-04-03 02:30:13,910 (WARNING) - django.request - Forbidden: /store/collections/
2024-04-03 02:30:13,919 (WARNING) - django.request - Bad Request: /store/collections/
2024-04-03 02:30:35,742 (WARNING) - django.request - Unauthorized: /store/collections/
2024-04-03 02:30:35,758 (WARNING) - django.request - Forbidden: /store/collections/
2024-04-03 02:30:35,768 (WARNING) - django.request - Bad Request: /store/collections/
2024-04-03 02:30:51,165 (WARNING) - django.request - Unauthorized: /store/collections/
2024-04-03 02:30:51,178 (WARNING) - django.request - Forbidden: /store/collections/
2024-04-03 02:30:51,187 (WARNING) - django.request - Bad Request: /store/collections/
2024-04-03 02:33:09,105 (WARNING) - django.request - Unauthorized: /store/collections/
2024-04-03 02:33:09,119 (WARNING) - django.request - Forbidden: /store/collections/
2024-04-03 02:33:09,129 (WARNING) - django.request - Bad Request: /store/collections/
2024-04-03 02:33:30,305 (WARNING) - django.request - Unauthorized: /store/collections/
2024-04-03 02:33:30,318 (WARNING) - django.request - Forbidden: /store/collections/
2024-04-03 02:33:30,330 (WARNING) - django.request - Bad Request: /store/collections/
2024-04-03 02:33:48,426 (WARNING) - django.request - Unauthorized: /store/collections/
2024-04-03 02:33:48,442 (WARNING) - django.request - Forbidden: /store/collections/
2024-04-03 02:33:48,460 (WARNING) - django.request - Bad Request: /store/collections/
2024-04-03 02:36:36,469 (WARNING) - django.request - Unauthorized: /store/collections/
2024-04-03 02:36:36,483 (WARNING) - django.request - Forbidden: /store/collections/
2024-04-03 02:36:36,493 (WARNING) - django.request - Bad Request: /store/collections/
2024-04-03 02:38:04,042 (WARNING) - django.request - Unauthorized: /store/collections/
2024-04-03 02:38:04,061 (WARNING) - django.request - Forbidden: /store/collections/
2024-04-03 02:38:04,071 (WARNING) - django.request - Bad Request: /store/collections/
2024-04-03 02:38:25,838 (WARNING) - django.request - Unauthorized: /store/collections/
2024-04-03 02:38:25,854 (WARNING) - django.request - Forbidden: /store/collections/
2024-04-03 02:38:25,867 (WARNING) - django.request - Bad Request: /store/collections/
2024-04-03 02:44:16,220 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-04 06:16:59,841 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-04 06:17:00,849 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-04 06:21:01,459 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-04 06:21:04,144 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2024-04-04 06:21:04,453 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-04 06:22:32,519 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-04 06:22:33,306 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-04 06:22:33,893 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-04 06:22:36,862 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2024-04-04 06:22:37,117 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-04 06:25:08,895 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-04 06:25:09,013 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-04 06:25:10,211 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-04 06:25:13,529 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2024-04-04 06:25:13,816 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-04 06:30:20,413 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-04 06:30:20,579 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-04 06:30:22,625 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-04 06:30:24,969 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2024-04-04 06:30:25,224 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-04 06:45:53,432 (INFO) - django.utils.autoreload - /app/storefront/settings/dev.py changed, reloading.
2024-04-04 06:45:54,899 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2024-04-04 06:45:55,261 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-04 06:46:30,025 (INFO) - django.utils.autoreload - /app/storefront/settings/common.py changed, reloading.
2024-04-04 06:46:31,309 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2024-04-04 06:47:22,810 (INFO) - django.utils.autoreload - /app/storefront/settings/dev.py changed, reloading.
2024-04-04 06:47:24,121 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2024-04-04 06:47:27,162 (INFO) - django.utils.autoreload - /app/storefront/settings/dev.py changed, reloading.
2024-04-04 06:47:28,378 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2024-04-04 07:25:25,348 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2024-04-04 07:44:41,554 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-04 07:44:41,792 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-04 07:44:42,455 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-04 07:44:48,571 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2024-04-04 07:44:48,893 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-04 07:44:56,814 (INFO) - django.utils.autoreload - /app/storefront/settings/common.py changed, reloading.
2024-04-04 07:44:58,931 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2024-04-04 07:44:59,384 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-04 07:45:22,993 (INFO) - django.utils.autoreload - /app/storefront/settings/common.py changed, reloading.
2024-04-04 07:45:24,445 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2024-04-04 07:45:25,034 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-04 07:45:31,590 (INFO) - django.utils.autoreload - /app/storefront/settings/common.py changed, reloading.
2024-04-04 07:45:33,156 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2024-04-04 07:45:33,477 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-04 07:46:02,413 (INFO) - django.utils.autoreload - /app/storefront/settings/dev.py changed, reloading.
2024-04-04 07:46:03,858 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2024-04-04 07:46:04,392 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 02:41:12,525 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 02:41:12,626 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 05:40:14,196 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 05:40:14,404 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 05:44:32,734 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 05:44:33,400 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 05:44:33,613 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 05:44:37,051 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2024-04-05 05:44:37,270 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 05:50:20,161 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 05:50:20,616 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 05:50:22,325 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 05:50:25,982 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2024-04-05 05:50:26,356 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 05:52:52,441 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 05:52:53,145 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 05:52:53,323 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 05:52:57,778 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2024-04-05 05:52:58,065 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 06:04:21,495 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 06:04:22,508 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 06:04:22,953 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 06:04:25,727 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2024-04-05 06:04:26,013 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 06:13:24,820 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 06:13:25,606 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 06:13:26,351 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 06:13:29,434 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2024-04-05 06:13:29,761 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 06:14:44,953 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 06:14:45,280 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 06:14:45,998 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 06:14:50,352 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2024-04-05 06:14:50,710 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 06:34:04,855 (INFO) - django.utils.autoreload - /app/storefront/settings/dev.py changed, reloading.
2024-04-05 06:34:06,502 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2024-04-05 06:34:06,872 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 06:34:15,506 (INFO) - django.utils.autoreload - /app/storefront/settings/dev.py changed, reloading.
2024-04-05 06:34:16,744 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2024-04-05 06:34:17,030 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 06:39:57,976 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 06:39:58,581 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 06:39:59,232 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 06:40:02,041 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2024-04-05 06:40:02,342 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 06:44:48,123 (INFO) - django.utils.autoreload - /app/storefront/settings/dev.py changed, reloading.
2024-04-05 06:44:49,571 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2024-04-05 06:44:49,994 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 06:44:52,502 (INFO) - django.utils.autoreload - /app/storefront/settings/dev.py changed, reloading.
2024-04-05 06:44:53,601 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2024-04-05 06:44:53,782 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 06:44:58,781 (INFO) - django.utils.autoreload - /app/storefront/settings/dev.py changed, reloading.
2024-04-05 06:44:59,783 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2024-04-05 06:44:59,979 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 06:45:03,989 (INFO) - django.utils.autoreload - /app/storefront/settings/dev.py changed, reloading.
2024-04-05 06:45:05,161 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2024-04-05 06:45:05,350 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 06:45:07,473 (INFO) - django.utils.autoreload - /app/storefront/settings/dev.py changed, reloading.
2024-04-05 06:45:08,627 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2024-04-05 06:45:08,832 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 06:46:36,255 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 06:46:36,914 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 06:46:37,180 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 06:46:39,946 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2024-04-05 06:46:40,213 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 06:53:06,203 (INFO) - django.utils.autoreload - /app/storefront/settings/dev.py changed, reloading.
2024-04-05 06:53:07,700 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2024-04-05 06:53:08,025 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 06:53:10,467 (INFO) - django.utils.autoreload - /app/storefront/settings/dev.py changed, reloading.
2024-04-05 06:53:11,675 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2024-04-05 06:53:11,883 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 07:02:10,503 (INFO) - django.utils.autoreload - /app/storefront/settings/dev.py changed, reloading.
05 07:01:48,869 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 07:02:10,423 (INFO) - django.utils.autoreload - /Users/ermiyasggod/Desktop/DjangoProject/storefront3/storefront/settings/dev.py changed, reloading.
2024-04-05 07:02:12,810 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2024-04-05 07:02:13,085 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 07:02:15,340 (INFO) - django.utils.autoreload - /app/storefront/settings/dev.py changed, reloading.
/storefront/settings/dev.py changed, reloading.
2024-04-05 07:02:17,221 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2024-04-05 07:02:17,600 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 07:02:19,882 (INFO) - django.utils.autoreload - /app/storefront/settings/dev.py changed, reloading.
/storefront/settings/dev.py changed, reloading.
2024-04-05 07:02:21,365 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2024-04-05 07:02:21,672 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 07:03:28,547 (INFO) - django.utils.autoreload - /app/storefront/settings/common.py changed, reloading.
07:02:39,469 (WARNING) - silk.profiling.profiler - Cannot execute silk_profile as silk is not installed correctly.
2024-04-05 07:03:28,701 (INFO) - django.utils.autoreload - /Users/ermiyasggod/Desktop/DjangoProject/storefront3/storefront/settings/common.py changed, reloading.
2024-04-05 07:03:30,136 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2024-04-05 07:03:30,646 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2024-04-05 07:03:41,167 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2024-04-05 07:03:46,923 (WARNING) - django.request - Not Found: /favicon.ico
2024-04-05 07:03:54,464 (WARNING) - rest_framework.fields - min_value in DecimalField should be Decimal type.
2024-04-05 07:04:07,171 (WARNING) - rest_framework.fields - min_value in DecimalField should be Decimal type.
2024-04-05 07:04:22,341 (WARNING) - rest_framework.fields - min_value in DecimalField should be Decimal type.
2024-04-05 07:04:31,590 (WARNING) - rest_framework.fields - min_value in DecimalField should be Decimal type.
2024-04-05 07:04:48,753 (WARNING) - rest_framework.fields - min_value in DecimalField should be Decimal type.
2024-04-05 07:04:48,797 (WARNING) - django.request - Method Not Allowed: /store/carts/
2024-04-05 07:04:55,171 (WARNING) - django.request - Unauthorized: /store/customers/
2024-04-05 07:04:59,547 (WARNING) - django.request - Unauthorized: /store/orders/
2024-04-05 07:05:06,385 (WARNING) - rest_framework.fields - min_value in DecimalField should be Decimal type.
2024-04-05 07:05:09,418 (INFO) - playground.views - Calling httpbin
2024-04-05 07:05:12,608 (INFO) - playground.views - Received the response
2024-04-05 07:26:25,890 (INFO) - playground.views - Calling httpbin
2024-04-05 07:26:28,900 (INFO) - playground.views - Received the response
2024-04-05 07:26:57,531 (WARNING) - django.request - Unauthorized: /auth/users/me/
2024-04-05 07:27:09,957 (WARNING) - django.request - Method Not Allowed: /auth/jwt/create/
2024-04-05 07:27:56,055 (WARNING) - django.request - Method Not Allowed: /auth/jwt/create
2024-04-05 07:28:28,785 (WARNING) - rest_framework.fields - min_value in DecimalField should be Decimal type.
2024-04-05 07:28:28,788 (WARNING) - rest_framework.fields - min_value in DecimalField should be Decimal type.
2024-04-05 07:28:28,792 (WARNING) - rest_framework.fields - min_value in DecimalField should be Decimal type.
2024-04-05 07:28:47,375 (WARNING) - rest_framework.fields - min_value in DecimalField should be Decimal type.
2024-04-05 07:28:47,390 (WARNING) - rest_framework.fields - min_value in DecimalField should be Decimal type.
2024-04-05 07:28:57,168 (WARNING) - rest_framework.fields - min_value in DecimalField should be Decimal type.
2024-04-05 07:28:57,169 (WARNING) - rest_framework.fields - min_value in DecimalField should be Decimal type.
2024-04-05 07:28:57,171 (WARNING) - rest_framework.fields - min_value in DecimalField should be Decimal type.
2024-04-05 07:28:57,174 (WARNING) - rest_framework.fields - min_value in DecimalField should be Decimal type.
2024-04-05 07:29:04,606 (WARNING) - rest_framework.fields - min_value in DecimalField should be Decimal type.
2024-04-05 07:29:04,622 (WARNING) - rest_framework.fields - min_value in DecimalField should be Decimal type.
2024-04-05 07:29:04,628 (WARNING) - rest_framework.fields - min_value in DecimalField should be Decimal type.
2024-04-05 07:29:08,473 (WARNING) - rest_framework.fields - min_value in DecimalField should be Decimal type.
2024-04-05 07:29:08,474 (WARNING) - rest_framework.fields - min_value in DecimalField should be Decimal type.
2024-04-05 07:29:08,478 (WARNING) - rest_framework.fields - min_value in DecimalField should be Decimal type.
2024-04-05 07:29:12,122 (WARNING) - rest_framework.fields - min_value in DecimalField should be Decimal type.
2024-04-05 07:29:12,124 (WARNING) - rest_framework.fields - min_value in DecimalField should be Decimal type.
2024-04-05 07:29:12,127 (WARNING) - rest_framework.fields - min_value in DecimalField should be Decimal type.
2024-04-05 07:29:12,203 (WARNING) - django.request - Not Found: /store/products/1001/
2024-04-05 07:29:43,103 (WARNING) - django.request - Not Found: /store/collections/1/
2024-04-05 07:29:48,104 (WARNING) - rest_framework.fields - min_value in DecimalField should be Decimal type.
2024-04-05 07:29:48,154 (WARNING) - django.request - Method Not Allowed: /store/carts/
2024-04-05 07:29:51,135 (WARNING) - rest_framework.fields - min_value in DecimalField should be Decimal type.
2024-04-05 07:30:09,782 (WARNING) - django.request - Bad Request: /store/orders/
2024-04-05 07:30:29,442 (ERROR) - django.request - Internal Server Error: /store/customers/
Traceback (most recent call last):
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/django/db/backends/mysql/base.py", line 75, in execute
    return self.cursor.execute(query, args)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/MySQLdb/cursors.py", line 179, in execute
    res = self._query(mogrified_query)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/MySQLdb/cursors.py", line 330, in _query
    db.query(q)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/MySQLdb/connections.py", line 261, in query
    _mysql.connection.query(self, query)
MySQLdb.IntegrityError: (1048, "Column 'user_id' cannot be null")

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/rest_framework/viewsets.py", line 124, in view
    return self.dispatch(request, *args, **kwargs)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/rest_framework/mixins.py", line 19, in create
    self.perform_create(serializer)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/rest_framework/mixins.py", line 24, in perform_create
    serializer.save()
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/rest_framework/serializers.py", line 208, in save
    self.instance = self.create(validated_data)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/rest_framework/serializers.py", line 989, in create
    instance = ModelClass._default_manager.create(**validated_data)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/django/db/models/query.py", line 658, in create
    obj.save(force_insert=True, using=self.db)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/django/db/models/base.py", line 814, in save
    self.save_base(
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/django/db/models/base.py", line 877, in save_base
    updated = self._save_table(
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/django/db/models/base.py", line 1020, in _save_table
    results = self._do_insert(
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/django/db/models/base.py", line 1061, in _do_insert
    return manager._insert(
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/django/db/models/query.py", line 1805, in _insert
    return query.get_compiler(using=using).execute_sql(returning_fields)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/django/db/models/sql/compiler.py", line 1822, in execute_sql
    cursor.execute(sql, params)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/debug_toolbar/panels/sql/tracking.py", line 235, in execute
    return self._record(super().execute, sql, params)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/debug_toolbar/panels/sql/tracking.py", line 160, in _record
    return method(sql, params)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/django/db/backends/utils.py", line 102, in execute
    return super().execute(sql, params)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/django/db/backends/utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/django/db/backends/mysql/base.py", line 75, in execute
    return self.cursor.execute(query, args)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/MySQLdb/cursors.py", line 179, in execute
    res = self._query(mogrified_query)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/MySQLdb/cursors.py", line 330, in _query
    db.query(q)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/MySQLdb/connections.py", line 261, in query
    _mysql.connection.query(self, query)
django.db.utils.IntegrityError: (1048, "Column 'user_id' cannot be null")
2024-04-05 07:33:19,940 (ERROR) - django.request - Internal Server Error: /store/customers/
Traceback (most recent call last):
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/django/db/backends/mysql/base.py", line 75, in execute
    return self.cursor.execute(query, args)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/MySQLdb/cursors.py", line 179, in execute
    res = self._query(mogrified_query)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/MySQLdb/cursors.py", line 330, in _query
    db.query(q)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/MySQLdb/connections.py", line 261, in query
    _mysql.connection.query(self, query)
MySQLdb.IntegrityError: (1048, "Column 'user_id' cannot be null")

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/rest_framework/viewsets.py", line 124, in view
    return self.dispatch(request, *args, **kwargs)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/rest_framework/mixins.py", line 19, in create
    self.perform_create(serializer)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/rest_framework/mixins.py", line 24, in perform_create
    serializer.save()
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/rest_framework/serializers.py", line 208, in save
    self.instance = self.create(validated_data)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/rest_framework/serializers.py", line 989, in create
    instance = ModelClass._default_manager.create(**validated_data)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/django/db/models/query.py", line 658, in create
    obj.save(force_insert=True, using=self.db)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/django/db/models/base.py", line 814, in save
    self.save_base(
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/django/db/models/base.py", line 877, in save_base
    updated = self._save_table(
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/django/db/models/base.py", line 1020, in _save_table
    results = self._do_insert(
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/django/db/models/base.py", line 1061, in _do_insert
    return manager._insert(
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/django/db/models/query.py", line 1805, in _insert
    return query.get_compiler(using=using).execute_sql(returning_fields)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/django/db/models/sql/compiler.py", line 1822, in execute_sql
    cursor.execute(sql, params)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/debug_toolbar/panels/sql/tracking.py", line 235, in execute
    return self._record(super().execute, sql, params)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/debug_toolbar/panels/sql/tracking.py", line 160, in _record
    return method(sql, params)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/django/db/backends/utils.py", line 102, in execute
    return super().execute(sql, params)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/django/db/backends/utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/django/db/backends/mysql/base.py", line 75, in execute
    return self.cursor.execute(query, args)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/MySQLdb/cursors.py", line 179, in execute
    res = self._query(mogrified_query)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/MySQLdb/cursors.py", line 330, in _query
    db.query(q)
  File "/Users/ermiyasggod/.local/share/virtualenvs/storefront3-SbeUFEq5/lib/python3.9/site-packages/MySQLdb/connections.py", line 261, in query
    _mysql.connection.query(self, query)
django.db.utils.IntegrityError: (1048, "Column 'user_id' cannot be null")
2024-04-05 07:44:32,237 (WARNING) - rest_framework.fields - min_value in DecimalField should be Decimal type.
2024-04-05 07:44:44,840 (WARNING) - django.request - Not Found: /core
2024-04-05 07:46:14,019 (WARNING) - rest_framework.fields - min_value in DecimalField should be Decimal type.
2024-04-05 07:46:25,840 (WARNING) - django.request - Not Found: /user
2024-04-05 07:49:49,728 (WARNING) - rest_framework.fields - min_value in DecimalField should be Decimal type.
2024-04-05 07:51:37,639 (WARNING) - django.request - Not Found: /tags
2024-04-05 07:51:58,917 (INFO) - playground.views - Calling httpbin
2024-04-05 07:52:01,777 (INFO) - playground.views - Received the response
2024-04-05 07:52:04,003 (WARNING) - django.request - Not Found: /playground
2024-04-05 07:52:09,224 (INFO) - playground.views - Calling httpbin
2024-04-05 07:52:12,081 (INFO) - playground.views - Received the response
2024-04-07 03:12:43,105 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2024-04-07 03:13:38,903 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2024-04-07 03:13:44,660 (WARNING) - django.request - Not Found: /favicon.ico
2024-04-07 03:14:11,445 (WARNING) - silk.model_factory - Response to request with pk 05dddde2-f19b-4152-aad0-0da9dcf116b1 has content type text/javascript but was unable to parse it
2024-04-07 03:14:14,948 (WARNING) - silk.model_factory - Response to request with pk 111fbbb0-13ca-45af-8179-80b278a842f9 has content type text/javascript but was unable to parse it
2024-04-07 03:17:51,984 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2024-04-07 03:18:22,012 (WARNING) - silk.model_factory - Response to request with pk 771e3059-bb17-4112-a0c2-bc1c69ea6201 has content type text/javascript but was unable to parse it
